internal/report/slo.go

```diff
@@ -1305,6 +1378,19 @@ ___
 		doc.WriteString("\n")
 	}
 
+	doc.WriteString("___\n")
+	doc.WriteString(`## Helper Rules Group
+
+The following table describes how the check IDs are distributed.
+
+| ID  | Description   |
+| --  | --            |
+| 00X[|A-Z] | Conformance result rules |
+| 01X[|A-Z] | Runtime, Infrastructure requirements, and known issues' rules |
+| 02X[|A-Z] | Result archive annomaly detector's rules |
+| 03X[|A-Z] | OpenShift object's rules |
```

Comment:
https://github.com/redhat-openshift-ecosystem/opct/pull/171 is introducing a new group, OPCT-04X*,  maybe it would be nice to add it here.